### PR TITLE
Add Education View Component to app

### PR DESCRIPTION
**Pull request by cubeydice**
Resolves issue #20 

Description:
- Adds an EducationView component
- Allows users to view existing education
- Submits a GET request to the /resume/education endpoint upon entering the app

Changes Made:
- Created EducationView.jsx and EducationView.css
- Sends a GET request with the form data to the server
- Updated App.js to include new view

Future Updates:
- Use redux to manage application state

image

### CODE REVIEW

**Comment by cubeydice** [comment]
@Pradyuman7 This PR is ready for your review. It does not resolve an existing issue, but allows for users to view existing education

**Comment by Pradyuman7** [comment]
Nice, could you create an issue for this PR, it would help us to track the work better

File: package.json [hunk 1]
```diff
@@ -2,12 +2,14 @@
   "name": "orientation-project-js",
   "version": "0.1.0",
   "private": true,
+  "proxy":"http://localhost:5000/",
```
Comment (Pradyuman7) [inline package.json]: Let's not add these in the main repo

File: src/components/education/EducationForm.jsx [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+                'Content-Type': 'application/json',
+            },
+            body: JSON.stringify(formData)
+        })
+        .then(response => response.json())
+        .then(data => {
+            alert("Education successfully added!")
+        })
+        .catch((error) => {
+            alert("Error: Education not added :(")
```
Comment (Pradyuman7) [inline src/components/education/EducationForm.jsx:39]: Nice use of alertbox, we can also console log the actual error

File: package.json (hunk 1 above)
Comment (cubeydice) [reply package.json]: Removed!

File: src/components/education/EducationForm.jsx (hunk 2 above)
Comment (cubeydice) [reply src/components/education/EducationForm.jsx:39]: Added console log of the error

**Comment by cubeydice** [comment]
@Pradyuman7 I updated for your comments and created a new issue #20

**Review by Pradyuman7 — APPROVED** [review]
LGTM 💯 

Can you resolve the conflicts, I'll merge then

**Comment by cubeydice** [comment]
@Pradyuman7 Conflicts resolved!

**Pull request merged**